<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>750</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/resource/IMG_20221003_153501.png</normaloff>:/resource/IMG_20221003_153501.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>18</width>
    <height>18</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="MyTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>25</height>
       </size>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>true</bool>
      </property>
      <widget class="FileWidget" name="tab">
       <attribute name="icon">
        <iconset theme="emblem-important">
         <normaloff>.</normaloff>.</iconset>
       </attribute>
       <attribute name="title">
        <string>Primary file view</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0" colspan="2">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string/>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QToolButton" name="toolButton">
             <property name="text">
              <string>&lt;-</string>
             </property>
             <property name="icon">
              <iconset resource="resource.qrc">
               <normaloff>:/resource/cdParent.png</normaloff>:/resource/cdParent.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButton_2">
             <property name="text">
              <string>-&gt;</string>
             </property>
             <property name="icon">
              <iconset resource="resource.qrc">
               <normaloff>:/resource/backChild.png</normaloff>:/resource/backChild.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit">
             <property name="minimumSize">
              <size>
               <width>500</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Jump to secondary view</string>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
             <property name="default">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="TreeView" name="treeView_3">
          <property name="focusPolicy">
           <enum>Qt::TabFocus</enum>
          </property>
          <property name="acceptDrops">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="icon">
        <iconset theme="edit-find">
         <normaloff>.</normaloff>.</iconset>
       </attribute>
       <attribute name="title">
        <string>Secondary file view</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0" colspan="2">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string/>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLineEdit" name="lineEdit_3"/>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_2">
             <property name="text">
              <string>Back to primary view</string>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
             <property name="default">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="TreeView" name="treeView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="icon">
        <iconset theme="edit-select-all">
         <normaloff>.</normaloff>.</iconset>
       </attribute>
       <attribute name="title">
        <string>Sort view</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout_3">
        <item row="0" column="0" colspan="2">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string/>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QComboBox" name="comboBox">
             <property name="minimumSize">
              <size>
               <width>500</width>
               <height>0</height>
              </size>
             </property>
             <property name="editable">
              <bool>true</bool>
             </property>
             <item>
              <property name="text">
               <string>pdf</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>zip</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>rar</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>png</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>cpp</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>jpg</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_3">
             <property name="text">
              <string>sort</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="ListWidget" name="listWidget_4">
          <property name="statusTip">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1500</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuview">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuhelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menufile"/>
   <addaction name="menuview"/>
   <addaction name="menuhelp"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="cursor">
    <cursorShape>UpArrowCursor</cursorShape>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetClosable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="TreeView" name="treeView_2">
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <property name="dragDropOverwriteMode">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="FindWidget" name="dockWidget_3">
   <property name="features">
    <set>QDockWidget::DockWidgetClosable</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QFormLayout" name="formLayout_5">
     <item row="0" column="0">
      <widget class="QComboBox" name="comboBox_2">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_5">
       <property name="text">
        <string>Browser</string>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QComboBox" name="comboBox_3">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>Find</string>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="TableWidget" name="tableWidget">
       <property name="frameShape">
        <enum>QFrame::VLine</enum>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <property name="verticalScrollMode">
        <enum>QAbstractItemView::ScrollPerPixel</enum>
       </property>
       <property name="showGrid">
        <bool>true</bool>
       </property>
       <property name="rowCount">
        <number>0</number>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>1</number>
       </attribute>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderCascadingSectionResizes">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderMinimumSectionSize">
        <number>0</number>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>30</number>
       </attribute>
       <attribute name="verticalHeaderHighlightSections">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderShowSortIndicator" stdset="0">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderStretchLastSection">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="exitmenu">
   <property name="text">
    <string>exit</string>
   </property>
  </action>
  <action name="actionabout">
   <property name="text">
    <string>about</string>
   </property>
  </action>
  <action name="actionnavigator">
   <property name="text">
    <string>navigator</string>
   </property>
  </action>
  <action name="actionnewfile">
   <property name="text">
    <string>newfile</string>
   </property>
  </action>
  <action name="actionnewdir">
   <property name="text">
    <string>newdir</string>
   </property>
  </action>
  <action name="actionfind">
   <property name="text">
    <string>find</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TreeView</class>
   <extends>QTreeView</extends>
   <header>treeview.h</header>
  </customwidget>
  <customwidget>
   <class>FileWidget</class>
   <extends>QWidget</extends>
   <header>filewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ListWidget</class>
   <extends>QListWidget</extends>
   <header>listwidget.h</header>
  </customwidget>
  <customwidget>
   <class>FindWidget</class>
   <extends>QDockWidget</extends>
   <header>findwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TableWidget</class>
   <extends>QTableWidget</extends>
   <header>tablewidget.h</header>
  </customwidget>
  <customwidget>
   <class>MyTabWidget</class>
   <extends>QTabWidget</extends>
   <header>mytabwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
